kano/network.py

```diff
@@ -26,16 +26,9 @@
 import re
 import shutil
 from kano.utils import run_cmd, get_user_unsudoed, run_bg, write_file_contents
+from kano import paths
```

Comment:
Doesn't this need to be

```python
from kano.paths import DNS_FILE, DNS_INTERFACES_FILE, \
    DNS_INTERFACES_BACKUP_FILE, SUPPLICANT_CMD, \
    SUPPLICANT_LOGFILE, SUPPLICANT_CONFIG, \
    INTERNET_UP_FILE, KANO_CONNECT_PIDFILE

Reply:
Tested my changeset on the kit and works just fine. I would prefer to avoid redundancy